ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/StatsUpdater.java

```diff
@@ -52,10 +52,6 @@ public final class StatsUpdater {
      */
     public void gatherStats(CompactionInfo ci, HiveConf conf, String userName, String compactionQueueName) {
         try {
-            if (!ci.isMajorCompaction()) {
```

Comment:
how much overhead we could get on a production cluster? AFAIK when multiple workers are used those would try to initiate a new Tez session.